### PR TITLE
Fixed param names for website operators

### LINKED ISSUE #1738: Documentation property SmartContent
the parameter `website_categories_operator` is actually `website_category_operator`,

the question is both in plural or not, so change in code or in documentation?
website_tags_operator
website_categories_operator

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [x] test coverage
- [ ] gather feedback for my changes
- [x] submit changes to the documentation

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1738 
| BC breaks        | yes but only to dev-develop (how should we handle that? /cc @danrot )
| Documentation PR | none (its already like that)

### CODE REVIEW

**Comment by danrot** [comment]
I think @chirimoya said last week that we don't have to mention develop BC breaks in the `UPGRADE.md` file.

File: src/Sulu/Component/SmartContent/ContentType.php [hunk 1]
```diff
@@ -259,13 +259,13 @@ public function getContentData(PropertyInterface $property)
...
         $filters['websiteTags'] = $this->tagRequestHandler->getTags($params['tags_parameter']->getValue());
-        $filters['websiteTagOperator'] = $params['website_tag_operator']->getValue();
+        $filters['websiteTagOperator'] = $params['website_tags_operator']->getValue();
 
         // extends selected filter with requested categories
         $filters['websiteCategories'] = $this->categoryRequestHandler->getCategories(
             $params['categories_parameter']->getValue()
         );
-        $filters['websiteCategoryOperator'] = $params['website_category_operator']->getValue();
+        $filters['websiteCategoryOperator'] = $params['website_categories_operator']->getValue();
```
Comment (danrot) [inline src/Sulu/Component/SmartContent/ContentType.php]: Shouldn't we also rename the two array keys in the `$filters` variable?

Comment (wachterjohannes) [inline src/Sulu/Component/SmartContent/ContentType.php]: your right i have overseen this ...

**Comment by danrot** [comment]
@wachterjohannes Now the tests are failing :wink:

**Pull request merged**